pod/perldiag.pod

```diff
@@ -657,12 +657,24 @@ is currently being compiled. Since this method is used to introduce new
 lexical subroutines into the scope currently being compiled, this is not
 going to have any effect.
 
-=item builtin::unimport can only be called at compile time
+=item builtin::unimport has now been removed
 
-(F) The C<unimport> method of the C<builtin> package was invoked when no code
-is currently being compiled. Since this method is used to remove
-previously-introduced lexical subroutines from the scope currently being
-compiled, this is not going to have any effect.
+(F) You can no longer remove a lexically-imported L<builtin> function by
+using the C<no builtin ...> syntax.
+
+Between development versions Perl 5.39.3 and Perl 5.39.7 it was possible
+to undo the effect of a C<use builtin ...> import by issuing a
+C<no builtin ...> directive.  This was found to be too complex in behaviour
+and introduce many subtle questions about the nature of hiding lexical
+functions, so was reomved in Perl 5.39.8.
+
+If you wish to limit the scope of visiblity of an imported C<builtin>
+function, put it inside its own scope:
+
+    {
+      use builtin 'refaddr';
+      ...
+    }
```

Comment:
Instead of mentioning that workaround here, it should be in the main `builtin` docs, where the `no builtin` was mentioned previously.

Reply:
done